### ISSUE TITLE
Compatibility with Alfresco Community 5.2

### ISSUE THREAD

**Issue by MrMXBryant** [Acosix/alfresco-simple-content-stores#38]
I tried to integrate the contentstore add-on with Community 5.2. I received Spring errors as 5.2 uses Spring 3 and the latest version of the add-on is Spring 5. Did you have to do any massaging to get the module to work with 5.2? Or should I drop down to the 5.0 add-on version?
Thanks in advance.

**Comment by AFaust** [comment]
This addon should not use any feature that requires Spring 5 - it is only linking to Spring 5 due to transitive dependencies imported from Alfresco. As stated in the global Readme, the addon should be compatible with 5.2 and above. Can you please provide the specific error(s) you are getting?

**Comment by MrMXBryant** [comment]
Axel, thanks so much for the quick response. I am developing on a separate network so I have to retype the error. 
It's a NoClassDefFoundError for org.springframework.beans.factory.support.BeanDefinitionRegistryPostProcessor Caused by CannotLoadBeanClassException: Error loading class [de.acosix.alfresco.utility.common.spring.PropertyAlteringBeanDefinitionRegistryPostProcessor] for bean with 'acosix-simple-content-stores-enablingBootstrap-subsystem' defined in URL [jar file: tomcat/shared/lib/de.acosix.alfresco.simplecontentstores.repo-1.2.0-SNAPSHOT-installable.jar! alfresco/module/acosix-simple-content-stores/module-context.xml]: problem with class file or dependent class: NoClassDefFoundError: org.springframework.beans.factory.support.BeanDefinitionRegistryPostProcessor

**Comment by MrMXBryant** [comment]
Axel, I am no intimately familiar with Alfresco setup but inherited a project that I have been maintaining. I created a new environment with a blank database to test implement and test encryption using your add-on. The error above started with creating a bean path.updateAdminUserWhendefault because I commented out the admin user password property, uncommented it and then received an error from your InitializingBean GroupNodeLocator which is leading me to believe there is supposed to be pre-existing data.  Is that the case?
